2_FirstComponent.md

```diff
@@ -67,9 +21,9 @@ If you recall from the introduction, a can.Component is like a self-contained, m
 - JS
 - CSS
 
-This is why we created a components folder for our app---instead of, say, a JS folder. Each component we develop will be in a folder that contains all the files that support that component. This makes components portable, enabling you to reuse them across projects. It also isolates them, making them easier to test and maintain.
+This is why we created a components directory for our app---instead of, say, a JS directory. Each component we develop will be in a directory that contains all the files that support that component. This makes components portable, enabling you to reuse them across projects. It also isolates them, making them easier to test and maintain.
```

Comment:
For some strange reason, "folder structure" threw me off. "directory structure" sounds more correct to me, but that's completely subjective.